splitgraph/engine/base.py

```diff
@@ -134,22 +135,42 @@ def copy_table(
         target_schema: str,
         target_table: str,
         with_pk_constraints: bool = True,
+        cursor_fields: Optional[Dict[str, str]] = None,
```

Comment:
![image](https://user-images.githubusercontent.com/889383/157889357-c115b363-6c76-4c48-ab48-94d133e1a0a0.png)


![image](https://user-images.githubusercontent.com/889383/157889137-8e2a5219-ec36-4bf3-a9c0-382ee7f4d241.png)

![image](https://user-images.githubusercontent.com/889383/157889168-9a13630f-6810-4472-a9d4-e744eea4dd27.png)

Array fields are not supported by dynamic forms. I suppose I should relax the error message